cpu/em_cpuidle.py

```diff
@@ -87,8 +87,9 @@ def read_from_device_tree(self):
         os.chdir('/proc/device-tree/ibm,opal/power-mgt')
         cmd_args = ['lsprop', 'ibm,cpu-idle-state-names']
         output_string = subprocess.Popen(
-            cmd_args, stdout=subprocess.PIPE).communicate()[0]
+            cmd_args, stdout=subprocess.PIPE).communicate()[0].decode("utf-8")
         output = re.findall('\"[a-zA-Z0-9_]+\"', output_string)
+
```

Comment:
empty line needed ?

Reply:
will remove this , will include this in last commit as  i need to rebase